skyportal/handlers/api/gcn.py

```diff
@@ -33,20 +33,25 @@
 Session = scoped_session(sessionmaker(bind=DBSession.session_factory.kw["bind"]))
 
 
-def post_gcnevent(payload, user_id):
+def post_gcnevent(payload, user_id, session=Session()):
```

Comment:
```suggestion
def post_gcnevent(payload, user_id, session):
```

I don't think a session should be instantiated like that.

Reply:
Yes good point. Fixed.